### PR TITLE
Basic Visual Regression Testing

### LINKED ISSUE #255: SOHO-7464: Visual Regression Testing
- Compare visuals on a baseline w/ https://www.npmjs.com/package/protractor-image-comparison
- Take one screen shot at each width: desktop, table, mobile, and test all themes if possible

- [x] Setup config for protractor image comparison 
- [x] Make some sample tests
- [x] Run the image tests on CI server, and provide instructions about how to create baseline images

### PR DESCRIPTION
**Pull request by d-g-h**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Adds visual regression testing to existing components.

**Related github/jira issue (required)**:
Closes #255 

**Steps necessary to review your pull request (required)**:
`npm run e2e:ci`, check Travis for build results, and history

### CODE REVIEW

File: docs/TESTING.md [hunk 1]
```diff
@@ -81,37 +81,69 @@ npm run e2e:local:debug
...
+- In another terminal, run the e2e test with the command below
+
+```sh
+npx -n=--inspect-brk protractor test/protractor.local.debug.conf.js
+```
+
 - In Chrome open `chrome://inspect` in a new tab.
 - Click on the 'Target' you will see generated under remote target
-- Hit go on the debugger
-- If interested in the Axe results you can view `res.violations` in the console
```
Comment (tmcconechy) [inline docs/TESTING.md]: Maybe we can keep this not or a note like it as its good information to me to know if debugging that type of test.

Comment (d-g-h) [reply docs/TESTING.md]: I was going to rewrite an updated set of instructions once we activate/revisit these tests again.

Comment (tmcconechy) [reply docs/TESTING.md]: ok sounds good. i guess it warrants a separate section

Comment (d-g-h) [reply docs/TESTING.md]: Yeah, might be a good idea to wrap in `util.* as well. That might reduce some lines of code, and change the debugging experience a bit.

**Pull request merged**